region/brinstar/pink/Pink Brinstar Hopper Room.json

```diff
@@ -534,37 +794,58 @@
       "note": "Using a power bomb against the left wall will kill the enemies and trigger the elevator."
     },
     {
-      "link": [2, 2],
-      "name": "Leave With Runway",
-      "requires": [],
-      "exitCondition": {
-        "leaveWithRunway": {
-          "length": 6,
-          "openEnd": 1
-        }
-      }
+      "link": [3, 1],
+      "name": "Dead Sidehoppers",
+      "requires": [
+        {"obstaclesCleared": ["A"]}
+      ]
     },
     {
-      "link": [2, 2],
-      "name": "Safe Kill",
+      "link": [3, 1],
+      "name": "Base",
       "requires": [
-        {"enemyKill": {
-          "enemies": [
-            ["Sidehopper", "Sm. Sidehopper", "Sm. Sidehopper"]
-          ],
-          "explicitWeapons": ["PowerBomb", "ScrewAttack", "Wave"]
-        }}
-      ],
-      "clearsObstacles": ["A"],
-      "note": "With access to the right side of the room, it's possible to kill the enemies safely with Wave or a PB or screw attack once on the other side."
+        {"or": [
+          {"and": [
+            "canCarefulJump",
+            "canCameraManip"
+          ]},
+          {"and": [
+            "ScrewAttack",
```

Comment:
With screw attack I would jump at the big hopper from above, and I usually don't get hit.
But it looks always safe to just jump down the left side while in screw.  Then jump again to kill everything.

Reply:
Looks like you can pretty much just jump anywhere as long as you spin jump again very quickly. I mentioned the left side, so the small one will hit you on a fail.